kie-api/src/main/java/org/kie/api/runtime/ConversationContext.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.api.runtime;
+
+public interface ConversationContext extends Context {
+    long getConversationId();
```

Comment:
shouldn't we have something more "advanced" for conversation id like UUID or similar so it might be more unique across jvms?